Rocket.Chat/Resources/pt-BR.lproj/Localizable.strings

```diff
@@ -154,6 +154,11 @@
 "subscriptions.list.no_message" = "Nenhuma mensagem";
 "subscriptions.list.sent_an_attachment" = "enviou um anexo";
 "subscriptions.list.date.yesterday" = "Ontem";
+"subscriptions.list.actions.read" = "Lido";
+"subscriptions.list.actions.unread" = "Não lido";
+"subscriptions.list.actions.favorite" = "Favorite"; // TODO
+"subscriptions.list.actions.unfavorite" = "Unfavorite"; // TODO
```

Comment:
"Remover dos Favoritos"

Reply:
This text is shown under the swipe actions. something shorter would be preferred.